lib/services/android-project-service.ts

```diff
@@ -417,24 +415,14 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 	}
 
 	public stopServices(projectRoot: string): Promise<ISpawnResult> {
-		let gradleBin = path.join(projectRoot, "gradlew");
-		if (this.$hostInfo.isWindows) {
-			gradleBin += ".bat";
-		}
-
-		return this.$childProcess.spawnFromEvent(gradleBin, ["--stop", "--quiet"], "close", { stdio: "inherit", cwd: projectRoot });
+		// TODO: Check if we have to pass compileSdk and other options here
```

Comment:
I've just tested it without args and it's working fine, so I'll remove the TODOD